hazelcast/test/src/HazelcastTests8.cpp

```diff
@@ -2054,6 +2054,115 @@ TEST(ClientMessageTest, testFragmentedMessageHandling)
     }
 }
 
+TEST(ClientMessageTest, test_encode_sql_query_id)
```

Comment:
FYI, I am not really familiar with the binary of client messages. So I have zero idea if test logic is ok not. Maybe someone familiar should check.